curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -162,12 +169,15 @@ FSStatusCode MdsClientImpl::UmountFs(const std::string& fsName,
     return ReturnError(rpcexcutor_.DoRPCTask(task, mdsOpt_.mdsMaxRetryMS));
 }
 
-FSStatusCode MdsClientImpl::GetFsInfo(const std::string& fsName,
-                                      FsInfo* fsInfo) {
+FSStatusCode MdsClientImpl::GetFsInfo(const std::string &fsName,
+                                      FsInfo *fsInfo) {
     auto task = RPCTask {
+        mdsClientMetric_.getFsInfo.qps.count << 1;
         GetFsInfoResponse response;
         mdsbasecli_->GetFsInfo(fsName, &response, cntl, channel);
+
         if (cntl->Failed()) {
+            mdsClientMetric_.getFsInfo.eps.count << 1;
```

Comment:
Should we cout eps when ret != FSStatusCode::OK like MountFs？

Reply:
ditto